spec/spec_helper.rb

```diff
@@ -5,9 +5,9 @@
 Coveralls.wear!
 
 SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::HTMLFormatter,
-  Coveralls::SimpleCov::Formatter
-])
+                                                                 SimpleCov::Formatter::HTMLFormatter,
+                                                                 Coveralls::SimpleCov::Formatter
+                                                               ])
```

Comment:
this looks like bad whitespace added?

Reply:
@ndushay It's to align the opening and closing square brackets.  It is the expected behavior of that particular cop.